samples/peripheral/radio_test/src/radio_test.c

```diff
@@ -94,21 +122,53 @@ static uint16_t channel_to_frequency(nrf_radio_mode_t mode, uint8_t channel)
 static nrf_radio_txpower_t dbm_to_nrf_radio_txpower(int8_t tx_power)
 {
 	switch (tx_power) {
```

Comment:
-7 -6 -4 -3 -2 -1 3 4 5 6 7 8 dBm are missing.
Also in `SHELL_STATIC_SUBCMD_SET_CREATE`.